packages/dashboard/src/pages/SignUp/steps/PersonaNameUI.tsx

```diff
@@ -0,0 +1,81 @@
+import { useEffect, useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { MaskTextField } from '@masknet/theme'
+import {
+    Body,
+    ColumnContentLayout,
+    Footer,
+    PersonaLogoBox,
+    SignUpAccountLogo,
+} from '../../../components/RegisterFrame/ColumnContentLayout'
+import { DashboardRoutes } from '@masknet/shared-base'
+import { Header } from '../../../components/RegisterFrame/ColumnContentHeader'
+import { Box, Button, Typography } from '@mui/material'
+import { useDashboardI18N } from '../../../locales'
+import { ButtonContainer } from '../../../components/RegisterFrame/ButtonContainer'
+
+const Label = ({ value }: { value: string }) => (
+    <Typography
+        variant="body2"
+        sx={{ marginBottom: '8px', fontWeight: 'bolder', color: (theme) => theme.palette.primary.main }}>
+        {value}
+    </Typography>
+)
+
+export interface PersonaNameUIProps {
+    error?: string
+    onNext(personaName: string): void
+}
+
+export const PersonaNameUI = ({ onNext, error }: PersonaNameUIProps) => {
+    const t = useDashboardI18N()
+    const navigate = useNavigate()
+
+    const [personaName, setPersonaName] = useState('')
+    const [helper, setHelper] = useState('')
+
+    useEffect(() => {
+        setHelper('')
+    }, [personaName])
+
+    useEffect(() => {
+        if (error) setHelper(error)
+    }, [error])
```

Comment:
```suggestion
    useEffect(() => {
        setHelper(error ?? '')
    }, [error, personaName])
```

Reply:
Seperate for this case: there is an error, need to reset helper when personaName changes